playbooks/replicate.yml

```diff
@@ -0,0 +1,86 @@
+- hosts: geniza_qa
+  connection: ssh
+  remote_user: pulsys
+  # override default location for db backup
+  vars:
+    - timestamp: "{{ ansible_date_time.date }}"
```

Comment:
I do use it! But I only noticed when I went looking that I *stashed* the tasks and variables for syncing media files instead of committing what I had, because it was still very rough.